pydss/flapper.py

```diff
@@ -93,10 +93,10 @@ def kick(self, lot, bid) :
 
         self.bids[bid_id].bid = bid;
         self.bids[bid_id].lot = lot;
-        self.bids[bid_id].guy = self.msg.sender;
-        self.bids[bid_id].end = self.now+self.tau
+        self.bids[bid_id].guy = sender;
+        self.bids[bid_id].end = now+self.tau
 
-        self.vat.move(self.msg.sender, self.address(self), lot);
+        self.vat.transferfrom(sender, self.ADDRESS, lot);
```

Comment:
my bad, could've been clearer - only replace instances of `gem.move` with `gem.transferFrom`, `vat.move` is a method we do have defined on the Vat class and should be called wherever necessary